gems/pending/Gemfile

```diff
@@ -18,7 +18,7 @@ gem "ezcrypto",                "=0.7",              :require => false
 gem "ffi",                     "~>1.9.3",           :require => false
 gem "ffi-vix_disk_lib",        "~>1.0.2",           :require => false  # used by lib/VixDiskLib
 gem "fog-openstack",           "~>0.1.7",           :require => false
-gem "hawkular-client",         "=2.3.0",            :require => false
+gem "hawkular-client",         "=2.4.0",            :require => false
```

Comment:
OK, I'll keep that in mind, hopefully :)  It's tough when only interacting sporadically with the MIQ eco-system... Thanks for the tips.